config/webpack/loaders/devSiteExample.js

```diff
@@ -1,34 +1,65 @@
 const path = require('path');
 const startCase = require('lodash.startcase');
+const findCss = require('./devSiteCssFinder');
 
 /**
  * Generate the example with the supplied file.
  * Don't use an arrow function
  */
-const loader = async function loader() {
+const loader = async function loader(content) {
   const callback = this.async();
 
   const exampleSource = this.resourcePath;
   const parsedResourcePath = path.parse(exampleSource);
+  const cssFileName = findCss(content);
 
   const code = [
     'import React from \'react\';',
     `import Example from '${exampleSource}';`,
     `import Code from '${exampleSource}?dev-site-codeblock';`,
     'import ExampleTemplate from \'terra-dev-site/lib/loader-components/_ExampleTemplate\';',
-    '',
-    `export default ({ title, description, isExpanded }) => (
+  ];
+
+  if (cssFileName !== undefined) {
+    try {
+      this.resolve('', cssFileName, async () => {
+      });
+      code.push(`import Css from '${cssFileName}?dev-site-codeblock';`,
+        `export default ({ title, description, isExpanded }) => (
+            <ExampleTemplate
+            title={ title || '${startCase(parsedResourcePath.name)}'}
+            description={description}
+            example={<Example />}
+            exampleCssSrc={<Css />}
+            exampleSrc={<Code />}
+            isExpanded={isExpanded}
+            />
+          );`);
+    } catch (err) {
+      code.push(`export default ({ title, description, isExpanded }) => (
+        <ExampleTemplate
```

Comment:
the code we push in the catch section looks equivalent to the last push. to make code more readable, we could update the try to return the code, make the catch only log an error, and remove the else.
```jsx
// pseudo code
try {
  resolve();
  code.push();
  return callback(code)
} catch (err) {
  log out error
}

code.push();
...

Reply:
Updated here: here: [d830ef0](https://github.com/cerner/terra-dev-site/pull/281/commits/d830ef07a820fb004bc59f65cde8f20ee7052353#).